content/02.main-text.md

```diff
@@ -24,11 +24,18 @@ What are the ideal features of an authoring/publishing platform?
 The benefits of openly licensed scholarly literature are so immense that we should avoid any platform whose incentives are misaligned with the open licensing of manuscripts.
 
 #. **Authoring and publishing should be free of charge.**
-Currently, article processing charges (and/or subscription revenue) do not fund the intellectual contributions or creation of manuscripts.
+Currently, article processing charges (APCs, and/or subscription revenue) do not fund the intellectual contributions or creation of manuscripts.
 Instead the revenue of publishers funds inefficient publishing platforms that can either be fully replaced by automation or the existing volunteer contributions of authors and reviewers.
-There will always be some cost for compute, hosting, and archiving of scholarly communication, but these should be in the range of dollars per article not thousands.
+Nonetheless, the cost of publishing continues to rise [@doi:10.18352/lq.10280].
+There will always be some cost for compute, hosting, and archiving of scholarly communication, but these should be in the range of dollars per article not thousands [@url:https://blog.joss.theoj.org/2019/06/cost-models-for-running-an-online-open-journal].
 Futhermore, several providers currently offer these services for free to public projects.
 
+Currently, journals do provide services that aren't entirely performed by unpaid volunteers.
```

Comment:
One thing I've noticed: you often do one word intro clauses (currently, nonetheless, however, instead). I think it might flow better without these in many cases.

Reply:
Currently, this is a bad habit. Nonetheless, I look forward to moderating it in the future. Only time will tell, however, whether I am successful. :smirk:

Reply:
😂